game.cpp

```diff
@@ -1,2 +1,37 @@
 #include <bits/stdc++.h>
-using namespace std;
\ No newline at end of file
+#include "main-character.cpp"
+#include "monster-generator.cpp"
+#include "battle.cpp"
+#include "store.cpp"
+
+using namespace std;
+
+const string INTRODUCTION = "BEM VINDO AO XIAQ\n";
+const string CONGRATULATIONS_MESSAGE = "Parabéns!! Você ganhou mais uma batalha..\n";
+const string OPTIONS_MESSAGE = "Digite 1 caso deseje ir a loja ou 1 se quiser continuar batalhando.\n";
+const string LOSS_MESSAGE = "Infelizmente chegamos ao fim após essa derrota.\n";
+
+void start_game(main_character character, monster enemy) {
+  //Select the difficult level before start the game.
+  while(battle_workflow(character, enemy)) {
```

Comment:
The battle workflow is the workflow of one single battle. The main method should keep the cicle: 4 simple battles, 1 battle with a boss, "Go to the store?"

Reply:
The way you are doing it's fighting the same monster all the time

Reply:
Also, after a battle_workflow if the player won, the monster game multipliers should be added to his, the same for coins